provision/openshift/isup.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# set -x
+
+source .env
+
+# kill all CrashLoopBackOff and ImagePullBackOff pods to trigger a fast restart and not wait Kubernetes
+kubectl get pods -n keycloak | grep -E "(BackOff|Error)" | tr -s " " | cut -d" " -f1 | xargs -r -L 1 kubectl delete pod -n keycloak
+
+MAXRETRIES=600
+
+declare -A SERVICES=( \
+ ["keycloak.${KC_HOSTNAME_SUFFIX}"]="realms/master/.well-known/openid-configuration" \
+)
+
+for SERVICE in "${!SERVICES[@]}"; do
+  RETRIES=$MAXRETRIES
+  # loop until we connect successfully or failed
+
+  if [ "${SERVICE}" == "keycloak-${KC_HOSTNAME_SUFFIX}" ]
```

Comment:
shouldn't this be `if [ "${SERVICE}" == "keycloak.${KC_HOSTNAME_SUFFIX}" ]` instead of `if [ "${SERVICE}" == "keycloak-${KC_HOSTNAME_SUFFIX}" ]`. Basically `.` instead of a `-` as defined in line 12 ?

Reply:
Thanks - fixed that in the next commit. This should prevent the symptoms you've seen earlier when "isup" was showing that a storage change was complete when it actually wasn't.